### PR TITLE
add SheetsManager.size

### PR DESCRIPTION
**Pull request by iamstarkov**
related https://github.com/cssinjs/react-jss/pull/123#discussion_r128999567

### CODE REVIEW

File: src/SheetsManager.js [hunk 1]
```diff
@@ -13,6 +13,10 @@ export default class SheetsManager {
 
   keys: Array<Object> = []
 
+  size() {
```
Comment (kof) [inline src/SheetsManager.js]: maybe as a getter?

Comment (kof) [inline src/SheetsManager.js]: would look more natural

Comment (iamstarkov) [inline src/SheetsManager.js]: sure, while im changing it. can you describe the differences?

Comment (iamstarkov) [inline src/SheetsManager.js]: nvm

Comment (iamstarkov) [inline src/SheetsManager.js]: @kof fixed

**Comment by iamstarkov** [comment]
@kof its a patch release, right?

**Comment by kof** [comment]
new api - minor

**Comment by iamstarkov** [comment]
@kof updated the changelog

**Comment by iamstarkov** [comment]
pls merge and release if you agree

File: changelog.md [hunk 2]
```diff
@@ -1,4 +1,8 @@
-## 8.1.0 / 2017-067-12
+## upcoming release
+
+- Added `SheetsManager.size()` method to get size of a `SheetsManager`
```
Comment (kof) [inline changelog.md]: getter, not method

File: tests/unit/SheetsManager.js [hunk 3]
```diff
@@ -84,4 +84,23 @@ describe('Unit: SheetsManager', () => {
       expect(detached).to.be(1)
     })
   })
+
+  describe('.size()', () => {
+    it('should be 0 from the start', () => {
+      const manager = new SheetsManager();
```
Comment (kof) [inline tests/unit/SheetsManager.js]: hmm, how did you manage to commit? lint-staged should have warned

**Comment by kof** [comment]
merged into next

**Pull request merged**